change/@fluentui-react-badge-bd7ce6fd-4495-46a0-9d19-6aa724e86cc7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 19 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26817&commitId=65e72531ffbdfbb7734521ba1c2e9744df59fc38&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1829 screenshots unchanged, 0 screenshots with different dimensions and 19 screenshots with visible difference.


  <details><summary><b>unknown</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge Converged.filled - High Contrast.chromium.png|1398| Changed|
|Badge Converged.outline - Dark Mode.chromium.png|2780| Changed|
|Badge Converged.outline - High Contrast.chromium.png|5561| Changed|
|Badge Converged.outline.chromium.png|3153| Changed|
|Badge Converged.size- extra-large - RTL.chromium.png|1879| Changed|
|Badge Converged.size- extra-large.chromium.png|1879| Changed|
|Badge Converged.size- extra-small - RTL.chromium.png|69| Changed|
|Badge Converged.size- extra-small.chromium.png|69| Changed|
|Badge Converged.size- large - RTL.chromium.png|1536| Changed|
|Badge Converged.size- large.chromium.png|1536| Changed|
|Badge Converged.size- medium - RTL.chromium.png|1389| Changed|
|Badge Converged.size- medium.chromium.png|1389| Changed|
|Badge Converged.size- small - RTL.chromium.png|1139| Changed|
|Badge Converged.size- small.chromium.png|1139| Changed|
|Badge Converged.size- tiny - RTL.chromium.png|39| Changed|
|Badge Converged.size- tiny.chromium.png|39| Changed|
|Badge Converged.tint - Dark Mode.chromium.png|965| Changed|
|Badge Converged.tint - High Contrast.chromium.png|3283| Changed|
|Badge Converged.tint.chromium.png|488| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>